apps/frontend/src/app/transformations-creator/transformations-creator.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <form [formGroup]="transformationForm" (ngSubmit)="createTransformation()">
+        <div class="col mb-3">
+          <label for="name" class="form-label">Name:</label><br />
+          <input id="name" class="form-control" type="text" formControlName="name" />
+          <div *ngIf="transformationForm.controls['name'].invalid && transformationForm.controls['name'].touched" class="alert alert-danger">
+            <div *ngIf="transformationForm.controls['name'].errors">
+              Name is required(Max. 40 characters)!
+            </div>
+          </div>
+        </div>
+        <div class="col mb-3">
+          <label for="description" class="form-label">Description:</label>
+          <textarea id="description" class="form-control" type="text" formControlName="description" rows="6"></textarea>
+        </div>
+        <div class="row">
+          <div class="col">
+            <button class="btn btn-primary" type="submit" [disabled]="!transformationForm.valid">Create</button><br/><br/>
```

Comment:
```suggestion
            <button class="btn btn-primary" type="submit" [disabled]="!transformationForm.valid">Create</button>
```

The line breaks can be removed, since they don't have any effect

Reply:
line break removed